meeteval/wer/__main__.py

```diff
@@ -444,6 +491,11 @@ def add_command(fn):
                     '--collar', type=positive_number,
                     help='Collar applied to the hypothesis timings'
                 )
+            elif name == 'regex':
```

Comment:
Does it make sense to add the option to all WERs and not just cpWER?

Reply:
Done